Makefile

```diff
@@ -454,6 +451,12 @@ update-dependencies: ## Uses go get -u to update all the dependencies while hold
 	# Copy everything to vendor directory
 	$(GO) mod vendor
 
+vet: ## Run mattermost go vet specific checks
+	env GO111MODULE=off $(GO) get -u github.com/mattermost/mattermost-govet
```

Comment:
My concern is that this will lead to at least one network request every time you run `make check-style`, which is often. What do you think about doing something like https://github.com/mattermost/mattermost-server/blob/master/Makefile#L156-L159 and requiring developers to install `mattermost-govet` them self?

Reply:
ok, works for me, my concern was about having it always up to date, but yes, probably keep this up-to-date in the CI is enough.

Reply:
Done